remoteappmanager/db/orm.py

```diff
@@ -0,0 +1,142 @@
+import contextlib
+
+from sqlalchemy import Column, Integer, Boolean, Unicode, ForeignKey
+from sqlalchemy.exc import OperationalError
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, relationship
+from sqlalchemy import create_engine
+
+from remoteappmanager.logging.logging_mixin import LoggingMixin
+
+Base = declarative_base()
+
+
+class UserTeam(Base):
+    """ The user (n <-> n) team association table """
+    __tablename__ = "user_team"
+    id = Column(Integer, primary_key=True)
+    user_id = Column(Integer, ForeignKey('user.id'))
+    team_id = Column(Integer, ForeignKey('team.id'))
```

Comment:
The majority of examples create the mapping using the `Table` class with the `Base.metadata`, instead of subclassing Base directly.  Haven't used sqlalchemy before, I would go with whatever the doc says or what others do.